validator_client/src/cli.rs

```diff
@@ -1,478 +1,466 @@
+                client. This will result in three (possibly four) epochs worth of missed \
+                attestations. If an attestation is detected during this period, it means it is \
+                very likely that you are running a second validator client with the same keys. \
+                This validator client will immediately shutdown if this is detected in order \
+                to avoid potentially committing a slashable offense. Use this flag in order to \
+                ENABLE this functionality, without this flag Lighthouse will begin attesting \
+                immediately.",
+        display_order = 0,
+        help_heading = FLAG_HEADER
+    )]
+    pub enable_doppelganger_protection: bool,
+
+    #[clap(
+        long,
+        alias = "private-tx-proposals",
+        help = "If this flag is set, Lighthouse will query the Beacon Node for only block \
+                headers during proposals and will sign over headers. Useful for outsourcing \
+                execution payload construction during proposals.",
+        display_order = 0,
+        help_heading = FLAG_HEADER
+    )]
+    pub builder_proposals: bool,
+
+    #[clap(
+        long,
+        value_name = "UNIX-TIMESTAMP",
+        help = "This flag takes a unix timestamp value that will be used to override the \
+                timestamp used in the builder api registration.",
+        display_order = 0
+    )]
+    pub builder_registration_timestamp_override: Option<u64>,
+
+    #[clap(
+        long,
+        value_name = "INTEGER",
+        default_value_t = 30_000_000,
+        help = "The gas limit to be used in all builder proposals for all validators managed \
+                by this validator client. Note this will not necessarily be used if the gas limit \
+                set here moves too far from the previous block's gas limit.",
+        display_order = 0
+    )]
+    pub gas_limit: u64,
+
+    #[clap(
+        long,
+        value_name = "BOOLEAN",
+        help = "Disables the service that periodically attempts to measure latency to BNs.",
+        display_order = 0,
+        help_heading = FLAG_HEADER
+    )]
+    pub disable_latency_measurement_service: bool,
+
+    #[clap(
+        long,
+        value_name = "INTEGER",
+        default_value_t = 500,
+        help = "Defines the number of validators per \
+                validator/register_validator request sent to the BN. This value \
+                can be reduced to avoid timeouts from builders.",
+        display_order = 0
+    )]
+    pub validator_registration_batch_size: usize,
+
+    #[clap(
+        long,
+        value_name = "UINT64",
+        help = "Defines the boost factor, \
+                a percentage multiplier to apply to the builder's payload value \
+                when choosing between a builder payload header and payload from \
+                the local execution node.",
+        display_order = 0
```

Comment:
Do we still need `.conflicts_with("prefer-builder-proposals")`?

Reply:
yes! i've re-addd that, thanks